src/include/distributed/distributed_planner.h

```diff
@@ -135,6 +135,24 @@ typedef struct RelationRowLock
 } RelationRowLock;
 
 
+/*
+ * Parameters to be set according to range table entries of a query.
+ */
+typedef struct RTEListProperties
```

Comment:
We could possibly store the same information in an `int` type where `RTEListProperties` could be a type. Something like the following:
```c
hasPostgresLocalTable 1 << 1
hasReferenceTable 1 << 2
...

```

This would have less storage, but this is just an optional alternative.